README.md

```diff
@@ -15,44 +21,113 @@
 
 [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A&style=flat-square)](https://ge.apollographql.com/scans)
 
-</div>
+  </div>
+</header>
+
+
+## ❓ Why Choose Apollo Kotlin?
+
+ ✅ Intuitive, powerful caching - intelligent in-memory or SQLite caching out of the box<br>
+ ✅ Excellent support for modern Android and Kotlin Multiplatform apps<br>
+ ✅ Always up-to-date - we prioritize support for the latest versions of Kotlin, Gradle, and more<br>
+ ✅ GraphOS ready - turnkey support for Persisted Queries, and `@defer`<br>
+ ✅ Production-tested - Powers countless apps worldwide that serve millions of end users<br>
+
+ ## 🚀 Quick Start
+ 
+If you are new to GraphQL, check out [the tutorial](https://www.apollographql.com/tutorials/apollo-kotlin-android-part1) that will guide you through building an Android app using Apollo.
+
+If you'd like to add Apollo Kotlin to an existing project, follow the steps below:
+
+Add the plugin to your `build.gradle.kts`:
+
+```kotlin
+plugins {
+  id("com.apollographql.apollo") version "x.x.x" // set to your current version
+}
+```
+
+Add the runtime dependency:
+
+```kotlin
+dependencies {
+  implementation("com.apollographql.apollo:apollo-runtime:x.x.x") // set to your current version
+}
+```
+
+Set the package name to use for the generated models:
+
+```kotlin
+apollo {
+  service("service") {
+    packageName.set("com.example")
+  }
+}
+```
+
```

Comment:
I have the feeling this either too short or too long. If we really want to include a quick start, it feels like it should explain how to write a query and use the generated models. A .gif like [SQLDelight](https://github.com/sqldelight/sqldelight) would do wonders